src/components/core/ScrollButton/ScrollButton.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Fade, useTheme } from '@material-ui/core';
+import { ReactComponent as ChevronUpIcon } from 'bootstrap-icons/icons/chevron-up.svg';
+import React, { useEffect, useState } from 'react';
+import Icon from '../Icon';
+import IconButton from '../IconButton';
+
+const ScrollButton = () => {
+  const [visible, setVisible] = useState(false);
+  const theme = useTheme();
+
+  const scrollToTop = () => {
+    window.scrollTo({
+      top: 0,
+      left: 0,
+      behavior: 'smooth',
+    });
+  };
+
+  useEffect(() => {
+    const toggleVisible = () => {
+      const scrolled = document.documentElement.scrollTop;
+      if (scrolled > 30) {
+        setVisible(true);
+      } else if (scrolled <= 30) {
+        setVisible(false);
+      }
```

Comment:
why `30` exactly ??
also can be simplified to `setVisible(scrolled > 30)`

Reply:
No reason at all.